lib/web/desktop_playback.go

```diff
@@ -30,15 +31,21 @@ func (h *Handler) desktopPlaybackHandle(
 	r *http.Request,
 	p httprouter.Params,
 	ctx *SessionContext,
+	site reversetunnel.RemoteSite,
 ) (interface{}, error) {
 	sID := p.ByName("sid")
 	if sID == "" {
 		return nil, trace.BadParameter("missing sid in request URL")
 	}
 
+	clt, err := ctx.GetUserClient(site)
```

Comment:
@russjones can you take a look at my PR description and confirm that I'm correctly understanding how `ctx.GetUserClient(site)` works here?